### ISSUE TITLE
Initialization options: "i18n.countryListAriaLabel" has no effect

### ISSUE THREAD

**Issue by Fectorez** [jackocnr/intl-tel-input#1521]
In the "i18n" object of initialization options, we can set `countryListAriaLabel`. However, when we set that field, the "aria-label" remains "List of countries" (the default value).

### Steps to reproduce
Create an HTML file with that content:
```js
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intl-tel-input@19.2.15/build/css/intlTelInput.css">
    <script src="https://cdn.jsdelivr.net/npm/intl-tel-input@19.2.15/build/js/intlTelInput.min.js"></script>

</head>
<body>
    <input type="tel" id="phone">
    
    <script>
        const input = document.querySelector("#phone");
        window.intlTelInput(input, {
          utilsScript: "https://cdn.jsdelivr.net/npm/intl-tel-input@19.2.15/build/js/utils.js",
          i18n: {
                selectedCountryAriaLabel: 'Pays sélectionné',
                countryListAriaLabel: 'Liste des pays',
                searchPlaceholder: 'Rechercher',
            }
        });
      </script>
</body>
</html>
```

### Expected behaviour
The related "aria-label" should be "Liste des pays":
```html
<ul class="iti__country-list" id="iti-0__country-listbox" role="listbox" aria-label="Liste des pays">
```

### Actual behaviour
The related "aria-label" is actually "List of countries":
```html
<ul class="iti__country-list" id="iti-0__country-listbox" role="listbox" aria-label="List of countries">
```

### What I noticed
When we open "https://cdn.jsdelivr.net/npm/intl-tel-input@19.2.15/build/js/intlTelInput.min.js", we can see that:
`selectedCountryAriaLabel` and `searchPlaceholder`, which are working, are literally present in that file. However, `countryListAriaLabel` is replaced by `mAriaLabel`.
In fact, if I use  `mAriaLabel` instead of `countryListAriaLabel`, everything is working.

### Initialisation options
```js
{
    utilsScript: "https://cdn.jsdelivr.net/npm/intl-tel-input@19.2.15/build/js/utils.js",
    i18n: {
          selectedCountryAriaLabel: 'Pays sélectionné',
          countryListAriaLabel: 'Liste des pays',
          searchPlaceholder: 'Rechercher',
      }
}
```

**Comment by jackocnr** [comment]
Good catch thanks. Fix released in v19.2.16.

**Comment by Fectorez** [comment]
> Good catch thanks. Fix released in v19.2.16.

Thank you for your reactivity!
